src/hu_nmt/data_augmentator/entrypoints/run_subj_obj_augmentation.py

```diff
@@ -55,6 +59,17 @@ def main(src_language, tgt_language, src_data_folder, tgt_data_folder, augmentat
                                            filter_same_pos_tag=filter_same_pos_tag,
                                            filter_for_noun_tags=filter_for_noun_tags)
 
+
+    if augmentation_type == 'ged' or augmentation_type == 'edge_mapper':
```

Comment:
can we give the old method a name as well and check for that? If it's none of the 3 valid options, let's throw an exception